loopchain/baseservice/rest_client.py

```diff
@@ -154,7 +154,9 @@ def _call_jsonrpc(self, target: str, method: RestMethod, params: Optional[NamedT
         url = self._create_jsonrpc_url(target, method)
         http_client = HTTPClient(url)
         request = self._create_jsonrpc_params(method, params)
-        return http_client.send(request, timeout=timeout)
+
+        response: Response = http_client.send(request, timeout=timeout)
+        return response.data.result
```

Comment:
What if response or response.data is None?

Reply:
Thank you for your precious review.
I have been tested in various situations and reached to an conclusion:
- `Response.data` is ensured if remote call is in success.

If failed by calling unknown method or exception raised in server side, client raises `ReceivedNon2xxResponseError`, which contains error code (404 or 500, ...)